lms/static/styles/lms.css

```diff
@@ -25,11 +25,11 @@ body {
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
+  justify-content: flex-start;
   height: 100%;
 }
```

Comment:
I wondered if we still need `display: flex` if we're not vertically centering this any more, but this still does horizontal centering without needing to have a fixed width. If you don't care about IE 11, you could use `margin: 0 auto; width: max-content;` to do the same thing.